src/Sylius/Bundle/AdminBundle/Menu/MainMenuBuilder.php

```diff
@@ -49,13 +49,24 @@ private function addCatalogSubMenu(ItemInterface $menu): void
         ;
 
         $catalog
-            ->addChild('taxons', ['route' => 'sylius_admin_taxon_create'])
+            ->addChild('taxons', ['route' => 'sylius_admin_taxon_create', 'extras' => ['routes' => [
+                ['route' => 'sylius_admin_taxon_create_for_parent'],
+                ['route' => 'sylius_admin_taxon_update'],
+            ]]])
```

Comment:
Couldn't you use the `::setExtra` method instead?

Reply:
I thought about that, but it doesn't work, I'm not sure why and haven't searched deeper 🤷🏻

Reply:
Could you recheck, because I just added
```php
->setExtra('routes', [
    ['route' => 'sylius_admin_taxon_create'],
    ['route' => 'sylius_admin_taxon_create_for_parent'],
    ['route' => 'sylius_admin_taxon_update'],
])
```
and it seems to work just fine.

Reply:
Ok, I see the difference, you are passing also the "main" route to extras, which I didn't do, so this approach requires duplication of one route in each menu item, now, I'm not sure if it looks better 😄